pkg/local_object_storage/metabase/delete.go

```diff
@@ -66,9 +72,11 @@ func (db *DB) Delete(prm DeletePrm) (DeleteRes, error) {
 	var rawRemoved uint64
 	var availableRemoved uint64
 	var err error
+	var sizes = make([]uint64, len(prm.addrs))
```

Comment:
do we always process these values on caller side? if not, as a slight optimization, we can make size collection optional

Reply:
> do we always process these values on caller side?

if metrics are turned off, that would be useless memuse, yes (but usually a stack mem, i guess). but i do not like the idea to provide an additional parameter (more complex code) for that. @roman-khimov

Reply:
We rarely run without metrics and I doubt it'll make any noticeable difference. So unless we have some real performance metrics suggesting a change, I'd stick to simpler code.